### ISSUE TITLE
Typescript 4.5 - sources always empty "Error running flat dts Failed to emit type definitions Error: Failed to emit type definitions"

### ISSUE THREAD

**Issue by natew** [run-z/rollup-plugin-flat-dts#19]
Getting this as of 4.5, I added logging inside to see why but nothing seems off it reads the right config.

```
Error running flat dts Failed to emit type definitions Error: Failed to emit type definitions
    at Object.writeOut (/Users/n8/dish/node_modules/rollup-plugin-flat-dts/dist/flat-dts.api.cjs:235:35)
```

For some reason this line:

```
const { options, errors, fileNames: files } = ts__default['default'].parseJsonConfigFileContent(config, ts__default['default'].sys, dirName);
```

Just returns an empty array always. Was working fine before.

**Comment by surol** [comment]
Thanks. Will look into this.

**Comment by surol** [comment]
Preliminary TypeScript-4.5 support added to [typescript-4.5](https://github.com/run-z/rollup-plugin-flat-dts/tree/typescript-4.5) branch.

Will release a `@next` version soon after TypeScript-4.5 enters [public beta](https://github.com/microsoft/TypeScript/issues/45418)

**Comment by surol** [comment]
Version 1.4.0-beta.0 released with support of TypeScript-4.5 added.
Please feel free to reopen this bug if it still isn't working for you.
